src/server/pps/cmds/cmds.go

```diff
@@ -824,6 +761,75 @@ you can increase the amount of memory used for the bloom filters with the
 	return result, nil
 }
 
+func handleCreatePipeline(metrics bool, reprocess bool, rebuild bool, pushImages bool, registry string, username string, password string, pipelinePath string, update bool) error {
+	cfgReader, err := ppsutil.NewPipelineManifestReader(pipelinePath)
+	if err != nil {
+		return err
+	}
+	client, err := pachdclient.NewOnUserMachine(metrics, true, "user")
+	if err != nil {
+		return fmt.Errorf("error connecting to pachd: %v", err)
+	}
+	defer client.Close()
+	for {
+		request, err := cfgReader.NextCreatePipelineRequest()
+		if err == io.EOF {
+			break
+		} else if err != nil {
+			return err
+		}
+		if request.Input.Atom != nil {
+			fmt.Fprintln(os.Stderr, "the `atom` input type is deprecated as of 1.8.1, please replace `atom` with `pfs`")
+		}
+		if update {
+			request.Update = true
+			request.Reprocess = reprocess
+		}
+		if rebuild || pushImages {
+			if rebuild && pushImages {
+				fmt.Fprintln(os.Stderr, "`--push-images` is redundant, as it's already enabled with `--rebuild`")
+			}
+			manager, err := newDockerManager(registry, username, password, request.Transform.Image)
+			if err != nil {
+				return err
+			}
+			if rebuild {
+				if request.Transform.Dockerfile == "" {
+					return fmt.Errorf("`dockerfile` must be specified in order to use `--rebuild`")
+				}
+				contextDir := request.Transform.WorkingDir
```

Comment:
This is conflating the meaning of `WorkingDir` in `Transform` it's used right now to control which directory usercode is run from (ie what it sees when it does `pwd`).